lib/Sabberworm/CSS/Value/URL.php

```diff
@@ -14,11 +14,21 @@ public function __construct(CSSString $oURL, $iLineNo = 0) {
 	}
 
 	public static function parse(ParserState $oParserState) {
-		$bUseUrl = $oParserState->comes('url', true);
+		$oParserState->setAnchor();
+		$sIdentifier = '';
+		for ($i = 0; $i < 3; $i++) {
+			$sChar = $oParserState->parseCharacter(true);
+			if ($sChar === null) {
+				break;
+			}
+			$sIdentifier .= $sChar;
+		}
+		$bUseUrl = $oParserState->streql($sIdentifier, 'url');
 		if ($bUseUrl) {
-			$oParserState->consume('url');
 			$oParserState->consumeWhiteSpace();
 			$oParserState->consume('(');
+		} else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method parse uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them.](https://app.codacy.com/manual/sabberworm/PHP-CSS-Parser/pullRequest?prid=4484968)